com.unity.render-pipelines.core/Runtime/Volume/Volume.cs

```diff
@@ -5,7 +5,7 @@ namespace UnityEngine.Rendering
     /// <summary>
     /// A generic Volume component holding a <see cref="VolumeProfile"/>.
     /// </summary>
-    [HDRPHelpURLAttribute("Volumes")]
+    [CoreRPHelpURL("Volumes", "com.unity.render-pipelines.high-definition")]
```

Comment:
wait, why does this one point to HDRP?

Reply:
yeah they moved the code but not the docs, at leas know is just an string that we can easily remove.

Reply:
If I remember well, there is history here.
Beyond memories, there was a time where no one remember Volume being in LightWeightRP (old name for UniversalRP).
Then a challenger appeared that change the face of RP world. Volume was introduced in this LWRP/URP too.
Though, both RP have really different components and both had there own documentation link.

Now:
Is the link working when in URP or does it redirect to unwanted documentation?